src/assets/scss/components/seriesTable.scss

```diff
@@ -0,0 +1,4 @@
+.thumbnail {
```

Comment:
I would suggest nesting `.thumbnail` inside other selectors to avoid future conflicts with class names.

As an example, maybe nest it inside `analylsisTableRow` class?
```suggestion
.analysisTableRow {
    .thumbnail {
```

Reply:
Done: https://github.com/darwinai/covidnet_ui/pull/17/commits/f5f7472bd15872d54f05fa4482aa0e64fdff9a64